packages/@react-aria/color/src/useColorArea.ts

```diff
@@ -68,44 +68,33 @@ export function useColorArea(props: AriaColorAreaProps, state: ColorAreaState, i
   let {keyboardProps} = useKeyboard({
     onKeyDown(e) {
       // these are the cases that useMove doesn't handle
-      if (!((e.shiftKey && /^Arrow(?:Right|Left|Up|Down)$/.test(e.key)) || /^(PageUp|PageDown|Home|End)$/.test(e.key))) {
+      if (!/^(PageUp|PageDown|Home|End)$/.test(e.key)) {
         return;
```

Comment:
Should we call `continuePropagation` on keys we don't handle to allow parents to handle them?